samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/TotalScoresBean.java

```diff
@@ -84,6 +91,15 @@
   public static final int CALLED_FROM_HISTOGRAM_LISTENER_STUDENT = 5;
   public static final int CALLED_FROM_EXPORT_LISTENER = 6;
   public static final int CALLED_FROM_NOTIFICATION_LISTENER = 7;
+
+  private static final SiteService siteService = (SiteService) ComponentManager.get(SiteService.class);
+  private static final ToolManager toolManger = (ToolManager) ComponentManager.get(ToolManager.class);
+  private static final ServerConfigurationService scs = (ServerConfigurationService) ComponentManager.get(ServerConfigurationService.class);
+
+  private static final String SAK_PROP_DELETE_RESTRICTED = "samigo.removeSubmission.restricted";
+  private static final boolean SAK_PROP_DELETE_RESTRICTED_DEFAULT = false;
+  private static final boolean RESTRICT_DELETE_GLOBAL = scs.getBoolean(SAK_PROP_DELETE_RESTRICTED, SAK_PROP_DELETE_RESTRICTED_DEFAULT);
```

Comment:
This seems like a CONSTANT but is not actually a constant

Reply:
Yes, you're right. I'll fix it up.